### PR TITLE
Added PeriodicEventManager

### PR DESCRIPTION
**Pull request by lxgr-linux**
Added:
- [x] `PeriodicEventManager`
- [x] Made water a periodic event
- [x] Extended `std_loop`

### CODE REVIEW

File: pokete_classes/landscape.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import random
+import logging
```
Comment (MaFeLP) [inline pokete_classes/landscape.py:2]: `logging` is never used, so why was it imported?

```suggestion
```

File: pokete_classes/loops.py [hunk 2]
```diff
@@ -10,14 +11,16 @@ def easy_exit_loop():
...
             return
         std_loop()
-        time.sleep(0.05)
 
 
-def std_loop(on_mvmp=True):
+def std_loop(on_mvmp=True, pevm=None):
     """Standart action executed in most loops
     ARGS:
         on_mvmp: Indicates if the loop is executed on movemap"""
```
Comment (MaFeLP) [inline pokete_classes/loops.py:19]: A description for `pevm` is missing in the docstring.

File: pokete_classes/loops.py [hunk 3]
```diff
@@ -10,14 +11,16 @@ def easy_exit_loop():
...
     if _ev.get() == "exit":
         raise KeyboardInterrupt
     if on_mvmp:
         notifier.next()
+    if pevm != None:
```
Comment (MaFeLP) [inline pokete_classes/loops.py:24]: Checking for `None` is better done with `is not None`:

```suggestion
    if pevm is not None:
```

File: pokete.py [hunk 4]
```diff
@@ -1001,9 +947,15 @@ def gen_obs():
...
+                "sand": Sand,
+                "meadow": Meadow,
+                "water": Water,
+            }[map_data[ob_map]["soft_obs"][soft_ob].get("cls", "meadow")]
             parse_obj(_map, soft_ob,
-                      Meadow(map_data[ob_map]["soft_obs"][soft_ob]["txt"],
-                             _map.poke_args),
+                      cls(map_data[ob_map]["soft_obs"][soft_ob]["txt"],
+                             _map.poke_args
+                                if cls != Water else _map.w_poke_args),
```
Comment (MaFeLP) [inline pokete.py:958]:
```suggestion
                          _map.poke_args
                          if cls != Water else _map.w_poke_args),
```

File: pokete_classes/periodic_event_manager.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
+from .settings import settings
+from .landscape import Meadow
+
```
Comment (MaFeLP) [inline pokete_classes/periodic_event_manager.py:3]:
```suggestion


```

**Review by MaFeLP — CHANGES_REQUESTED** [review]

File: pokete_classes/landscape.py (hunk 1 above)
Comment (lxgr-linux) [reply pokete_classes/landscape.py:2]: For debugging.

**Review by MaFeLP — APPROVED** [review]

**Pull request merged**